src/util/isReactFragment.tsx

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react';
+
+export const isReactFragment = (node: React.ReactNode) => {
```

Comment:
Should this be
```
export const isReactFragment = (node?: React.ReactNode) => {
```
considering the next line
```
if (!node) return false;
```
?

Reply:
I ported this over from another project we used this on. I'm alright with this if that is needed. Wasn't seeing any errors around it. But I'm using it so the `ButtonDropdown` can use a fragment as well and the keyboard functionality won't break since I'm using a slot for the `ProjectCard` `DropdownMenuItem`s